sections/node.zsh

```diff
@@ -20,12 +20,18 @@ SPACESHIP_NODE_COLOR="${SPACESHIP_NODE_COLOR="green"}"
 # ------------------------------------------------------------------------------
 
 # Show current version of node, exception system.
-spaceship_node() {
+spaceship_async_job_load_node() {
   [[ $SPACESHIP_NODE_SHOW == false ]] && return
-
   # Show NODE status only for JS-specific folders
   [[ -f package.json || -d node_modules || -n *.js(#qN^/) ]] || return
 
+  async_job spaceship spaceship_async_job_node "$PWD" "$PATH"
+}
+
+spaceship_async_job_node() {
+  [[ $SPACESHIP_NODE_SHOW == false ]] && return
+
+	PATH=$2
```

Comment:
I'll look into saving PATH and PWD for all sections, so that we don't have to remember to save and restore these variables all the time.

Reply:
The last commit on `master` https://github.com/maximbaz/spaceship-prompt/commit/c140e4405b9eac5811f66fb1ade3bc562be519cf should fix the issue with PATH not being available, could you please try and confirm, and update to not pass the PATH in the job?

The maintainer of zsh-async agreed to fix PWD, so that we don't have to remember to `cd` in the folder every time, but we can merge your change now and I'll remove the `cd` once it becomes obsolete.